### DISCUSSION TITLE
Failing installation of Cypress 9.5.3

### DISCUSSION THREAD

**Discussion by K00243724** [cypress-io/cypress#20959] [Questions and Help]
Hi, so I'm very new to Cypress and automation testing and still Trying after multiple attempts to install Cypress.
So I have **Node.js 16.13.2**
I've also have **npm 8.4.1**
installed. So after multiple attempts I deleted the project folder and made another. Now when I try to install Cypress it hangs on the first line....
![image](https://user-images.githubusercontent.com/46998407/162183259-9bb0db33-e7e9-4edf-8734-bee7d83fbd36.png)
Can I delete from AppData, Cypress folders in Local and Roaming folders, clearing the cache. 

I'm trying to install on Windows 10 and using a proxy.

**Comment by K00243724** [comment]
Still trying to fix and now I'm getting this 
![image](https://user-images.githubusercontent.com/46998407/162917774-30dc2cc8-c6bd-4f09-bc8d-6a33ca423780.png)
When I install Cypress it will install on the Project folder but only creates the Cypress folder and cache subfolder in the AppData/local/Cypress folder but does not install cypress executable .

Any suggestions please this problem is running over a week for me now. It's crazy, this should be a straight forward install!

**Comment by muratkeremozcan** [comment]
How is 9.5.4?
